tests/fail/parsingBoth/chars cannot be outside a subset of the ascii charset.mm0

```diff
@@ -0,0 +1,6 @@
+-- lexeme ::= symbol | identifier | number | math-string
+-- symbol ::= '*' | '.' | ':' | ';' | '(' | ')' | '>' | '{' | '}' | '=' | '_'
+-- identifier ::= [a-zA-Z_][a-zA-Z0-9_]*
+-- number ::= 0 | [1-9][0-9]*
+-- math-string ::= '$' [^$]* '$'
```

Comment:
Reading this test I don't see what the focus is. The comment should point out what is wrong in the test, it shouldn't just be a quote from the spec.

Also for some basic parsing tests you may want to not have any comments, because the comment parsing itself might be part of the problem.